File: src/main/java/com/team3/gudit/GuditApplication.java
package com.team3.gudit;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

@EnableScheduling
@SpringBootApplication
public class GuditApplication {
    public static void main(String[] args) {
        SpringApplication.run(GuditApplication.class, args);
    }
}

File: src/main/kotlin/com/team3/gudit/GuditApplication.kt
package com.team3.gudit

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
class GuditApplication
fun main(args: Array<String>) {
    runApplication<GuditApplication>(*args)
}